src/main/java/com/example/bookstoreapp/controller/BookController.java

```diff
@@ -33,8 +37,21 @@ public ResponseEntity<BookDto> getBookById(@PathVariable Long id) {
     }
 
     @PostMapping
-    public ResponseEntity<BookDto> createBook(@RequestBody CreateBookRequestDto bookDto) {
+    public ResponseEntity<BookDto> createBook(@RequestBody BookRequestDto bookDto) {
         BookDto createdBook = bookService.save(bookDto);
         return ResponseEntity.ok(createdBook);
     }
+
+    @PutMapping("/{id}")
+    public ResponseEntity<BookDto> updateBook(@PathVariable Long id, @RequestBody BookDto bookDto) {
```

Comment:
I suggest you to create UpdateBookDto or something like that and accept it here as a RequestBody